packages/client/src/events/callEventHandlers.ts

```diff
@@ -66,11 +66,6 @@ export const registerEventHandlers = (call: Call, dispatcher: Dispatcher) => {
     handleRemoteSoftMute(call),
   ];
 
-  if (call.ringing) {
-    // these events are only relevant when the call is ringing
-    eventHandlers.push(registerRingingCallEventHandlers(call));
-  }
-
```

Comment:
this part is unnecessary, as it was only creating duplicate ringing subscriptions as we do this now in registerRingingCallEffects()